### DISCUSSION TITLE
Evaluate a `:class` passing a function with args?

### DISCUSSION THREAD

**Discussion by tiago-pires** [alpinejs/alpine#1241] [1. Help]
I understand we can use a method returning a true or false for `:class`, but is it possible to pass arguments to this function?  
```
<div :class="{'some-class': foo(arg)}">...</div>
```

**Comment by KevinBatdorf** [comment]
It should work, but the best way to find out would be to try it.

**Reply by tiago-pires** [reply]
Yeah, it does:
```html
      <div x-data="{foo: x => x === 'a' ? true : false}">
         <button :class="{'myClass': foo('a') }">Click</button> 
      </div>
```

I had another bug in my code. But you are right, I should have tried it in isolation.
